custom_components/givenergy_local/dashboard.py

```
@@ -0,0 +1,447 @@
+            ("remaining_capacity", "Remaining Capacity"),
+            ("calibrated_capacity", "Calibrated Capacity"),
+            ("design_capacity", "Design Capacity"),
+            ("cell_voltages_sum", "Cell Voltages Sum"),
+            ("cell_count", "Cell Count"),
+        ],
+    )
+    cells = _bat_entity_rows(serial, [(f"cell_{i}_voltage", f"Cell {i}") for i in range(1, 17)])
+    temps = _bat_entity_rows(
+        serial,
+        [
+            ("cells_1_4_temperature", "Cells 1–4"),
```

Ruff (0.15.12)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)